bundles/org.openhab.binding.konnected/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -4,4 +4,5 @@
 	xsi:schemaLocation="https://openhab.org/schemas/binding/v1.0.0 https://openhab.org/schemas/binding-1.0.0.xsd">
 	<name>Konnected Binding</name>
 	<description>This is the binding for Konnected.</description>
+	<author>Zachary Christiansen</author>
```

Comment:
This tag is deprecated, please remove this.

Reply:
Removed the author tag.